etna/packages/etna-js/actions/magma_actions.js

```diff
@@ -357,8 +359,10 @@ export const requestAnswer = (question, callback = null) => dispatch => {
       if (callback) callback(response);
       else return response;
   }).catch( error => {
-    console.log(error);
-    throw new Error(error);
+    return Promise.resolve(error).then((e) => {
```

Comment:
Also very confused about this -- why is this extra layering necessary?

Reply:
I actually might have added this before making the change to `showError` above for the TSV download, when trying to figure out why the TSV download was throwing an exception. Might not be necessary -- will check.